qualification/pytest-jenkins-karoo.ini

```diff
@@ -22,4 +22,5 @@ max_antennas = 80
 wideband_channels = 1024 4096 8192 32768
 narrowband_channels = 32768
 narrowband_decimation = 8 16
+vlbi_decimation = 8
```

Comment:
Need to add it to pytest-jenkins-lab.ini as well. Also check if the documentation has an example ini file that needs updating - I seem to recall we removed the example and just referred the reader to the actual ini files for inspiration.

Reply:
Noted.

Reply:
For the record you were quite right Re:
> I seem to recall we removed the example and just referred the reader to the actual ini files for inspiration.

https://katgpucbf.readthedocs.io/en/latest/qualification.html#configuration